tests/wpunit/PluginObjectQueriesTest.php

```diff
@@ -57,7 +57,7 @@ public function testPluginQuery() {
 					'id' => $global_id,
 					'name' => 'Hello Dolly',
 					'pluginUri' => 'http://wordpress.org/plugins/hello-dolly/',
-					'version' => '1.6',
+					'version' => '1.7',
```

Comment:
we need to change the assertion we're making here so this doesn't cause failures when we test on different versions of WordPress core that ship with different versions of this plugin. I'll create another issue to address this.

Reply:
I'll write a helper function in the test